### PR TITLE
Use Akka 2.6.0-M1

### PR DESCRIPTION
**Pull request by marcospereira**
Use the first milestone for Akka 2.6.

### CODE REVIEW

File: core/play/src/test/java/play/mvc/RangeResultsTest.java [hunk 1]
```diff
@@ -280,12 +280,12 @@ public void shouldNotReturnRangeResultForSourceWhenHeaderIsNotPresent() throws I
   }
 
   @Test
-  public void shouldReturnRangeResultForSourceWhenHeaderIsPresentAndContentTypeWasSpecified() throws IOException {
+  public void shouldReturnRangeResultForSourceWhenHeaderIsPresentAndContentTypeWasSpecified()
```
Comment (marcospereira) [inline core/play/src/test/java/play/mvc/RangeResultsTest.java]: We need to check `test:javafmt` in our build. 🤷‍♂

File: scripts/scriptLib [hunk 2]
```diff
@@ -19,7 +19,7 @@ AKKA_HTTP_VERSION=""
 # Check if it is a scheduled build
 if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
     # `sort` is not necessary, but it is good to make it predictable.
-    AKKA_VERSION=$(curl -s https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.12/ | grep -oEi '2\.5-[0-9]{8}-[0-9]{6}' | sort | tail -n 1)
+    AKKA_VERSION=$(curl -s https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.12/ | grep -oEi '2\.6-[0-9]{8}-[0-9]{6}' | sort | tail -n 1)
```
Comment (marcospereira) [inline scripts/scriptLib]: Also starting to use 2.6 snapshots in our nightly builds.

File: core/play/src/test/java/play/mvc/RangeResultsTest.java (hunk 1 above)
Comment (marcospereira) [reply core/play/src/test/java/play/mvc/RangeResultsTest.java]: See #9240 for a fix.

**Comment by ignasi35** [comment]
I've blocked merge until https://github.com/playframework/playframework/pull/9240 is merged so this PR is cleaner.

**Comment by marcospereira** [comment]
Rebase against the master branch, so I've removed `status:block-merge` label.

**Comment by ignasi35** [comment]
nice!

**Pull request merged**